src/lib/support/tests/TestPrivateHeap.cpp

```diff
@@ -151,10 +150,10 @@ void FreeMergePreviousAndNext(nlTestSuite * inSuite, void * inContext)
     void * p2 = allocator.HeapAlloc(16);
     void * p3 = allocator.HeapAlloc(16);
 
-    NL_TEST_ASSERT(inSuite, nullptr != p1);
-    NL_TEST_ASSERT(inSuite, nullptr != p2);
-    NL_TEST_ASSERT(inSuite, nullptr != p3);
-    NL_TEST_ASSERT(inSuite, nullptr == allocator.HeapAlloc(1));
+    EXPECT_NE(nullptr, p1);
```

Comment:
If pointer is dereferenced after NULL check, use `ASSERT_*`. Fix it in all other places as well.

Reply:
Done!